prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -633,4 +643,36 @@ class ReportFunction(
             }
         }
     }
+
+    private fun handleAckRequest(request: HttpRequestMessage<String?>): HttpResponseMessage? {
+        // why does Azure handle Headers case-sensitive???
```

Comment:
I'm fine leaving this here because Azure does sound silly. But did you mean to leave it here?

Reply:
We should change the comment to the following if we leave it in (something more "professional"):

`// Azure headers are case-sensitive`